acceptance/tests/ticket_6928_puppet_master_parse_fails.rb

```diff
@@ -11,7 +11,7 @@
 
 step "Master: use --parseonly on an invalid manifest, should return 1 and issue deprecation warning"
 on master, puppet_master( %w{--parseonly /tmp/bad.pp} ), :acceptable_exit_codes => [ 1 ]
-  assert_match(/--parseonly has been removed. Please use \'puppet parser validate <manifest>\'/, stdout, "Deprecation warning not issued for --parseonly on #{master}" ) unless agent['locale'] == 'ja'
+  assert_match(/--parseonly has been removed. Please use \'puppet parser validate <manifest>\'/, stdout, "Deprecation warning not issued for --parseonly on #{master}" ) unless master['locale'] == 'ja'
```

Comment:
I can't think of what else to do except the loop. At least if we add it we'll be consistent I guess.